src/storage/dictionaries/clean.py

```diff
@@ -12,10 +12,13 @@
 f = open(file_name)
 f2 = open("modified.txt", "w")
 for line in f:
+    line = line.strip()
```

Comment:
All of our words when opened in file mode had spaces in them, this messed up my analysis of them which led to some single chars (`a`) being included because their length was actually > 2 due to having multiple spaces